### PR TITLE
docs: Update required node version in README

### PR DESCRIPTION
**Pull request by legodude17**
Ref #20425.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Much more info available via `npm help` once it's installed.
 
 ## IMPORTANT
 
-**You need node v4 or higher to run this program.**
+**You need node v6 or higher to run this program.**
```
Comment (nfischer) [inline README.md:14]: The text below references "v0.12", but should probably be changed to "v5"

Comment (legodude17) [reply README.md:14]: @nfischer Done! Thanks for the heads-up!

**Review by nfischer — APPROVED** [review]
LGTM, thanks!

**Comment by legodude17** [comment]
Travis build failing is due to `Object.entries` bug.

**Review by zkat — APPROVED** [review]

**Comment by zkat** [comment]
Thanks, @legodude17!

**Comment by legodude17** [comment]
🎉

**Pull request merged**